bin/ssh_scan_api

```diff
@@ -22,6 +22,18 @@ opt_parser = OptionParser.new do |opts|
     exit
   end
 
+  opts.on("--https", "Use HTTPS") do
+    options[:https] = true
+  end
```

Comment:
Does this just serve a self-signed cert if the key and crt files aren't provided?